lib/seek/json_metadata/serialization.rb

```diff
@@ -18,9 +18,13 @@ def data
       end
 
       def get_attribute_value(attr)
-        attr = attr.accessor_name if attr.is_a?(attribute_class)
-
-        data[attr.to_s]
+        if attr.try(:sample_attribute_type) && attr.sample_attribute_type.base_type == Seek::Samples::BaseType::LINKED_CUSTOM_METADATA
```

Comment:
you can use `linked_custom_metadata?` here, e.g `attr.try(:sample_attribute_type).try(:linked_custom_metadata?)`

Reply:
also, what if `get_attribute_value `is called for linked_custom_metadata but `attr `is a string ?

Reply:
> also, what if get_attribute_value is called for linked_custom_metadata but attr is a string ?


when `get_attribute_value` is called by `show` action, the value output is
```
attr=>#<CustomMetadataAttribute id: 286, custom_metadata_type_id: 16, sample_attribute_type_id: 26, required: false, pos: nil, title: "a_3_3_3", sample_controlled_vocab_id: nil, description: nil, label: nil, linked_custom_metadata_type_id: 15>
attr.linked_custom_metadata?=>true
value=>#<CustomMetadata id: 14, json_metadata: "{\"a_3_3_3_1\":\"3331\",\"a_3_3_3_2\":\"3332\",\"a_3_3_3_3\"...", item_type: nil, item_id: nil, custom_metadata_type_id: 15, custom_metadata_attribute_id: 286>
```

when `get_attribute_value` is called by `edit` action, the value output is
```
attr=>"a_3_3_3"
self=>#<CustomMetadata id: 13, json_metadata: "{\"a_3_3_1\":\"331\",\"a_3_3_2\":\"332\",\"a_3_3_3\":14}", item_type: nil, item_id: nil, custom_metadata_type_id: 16, custom_metadata_attribute_id: 296>
value=>14
```